adapta/storage/delta_lake/README.md

```diff
@@ -1,8 +1,8 @@
 # Delta Lake Operations
 
 Supported API:
-- read delta table as `pandas.DataFrame`
-- read delta table in batches of a provided size, each batch being `pandas.DataFrame`
+- read delta table as `MetaFrame` which can easily be converted to `pandas.DataFrame` or `polars.DataFrame`
+- read delta table in batches of a provided size, each batch being `MetaFrame`
```

Comment:
This will cause a mayhem in all systems. Default behaviour for all methods should be `pandas`, then users can opt-in into `MetaFrame`

Reply:
Now separated into v2 and v3